rocky/onboarding/views.py

```diff
@@ -340,8 +344,13 @@ def post(self, request, *args, **kwargs):
 
     def get_ooi_pks(self) -> list[str]:
         ooi = self.get_ooi(self.request.GET.get("ooi", ""))
-        hostname_ooi = [Hostname(name=ooi.web_url.tokenized["netloc"]["name"], network=ooi.network)]
-        return [hostname_ooi[0].primary_key]
+        try:
+            hostname_ooi = [Hostname(name=ooi.web_url.tokenized["netloc"]["name"], network=ooi.network)]
+            return [hostname_ooi[0].primary_key]
+        except AttributeError as error:
+            logger.error("Attribute error: %s", error)
+            messages.error(self.request, _("DNS Report cannot be created, no Hostname."))
+            return []
```

Comment:
I don't think this is a good fix for #4483 

The error states that `ooi.web_url` is `None`, therefore you should check for `None` instead of relying on attribute access.

`AttributeError`s are usually reflect bugs in the code rather than expected runtime conditions. It's better to address the root cause, which is an empty/ non-set `web_url` attribute